internal/driver/driver.go

```diff
@@ -177,7 +177,7 @@ func applyCommandOverrides(cmd string, outputFormat int, v variables) variables
 	}
 
 	if outputFormat == report.Proto || outputFormat == report.Raw {
-		trim, tagfilter, filter = false, false, false
+		trim = false
```

Comment:
Should we also stop resetting the filters for the peek command? We would be able to remove the whole filter / tagfilter machinery form the applyCommandOverrides function then. I can't think of a good reason to keep resetting the filters for the peek command.

Reply:
+1 to resetting it for peek as well.

The original intent was for peek to provide full details no matter what the settings were (to inspect at a very low level), but I think it is just confusing, and the user can always reset their filters if they so desire.

Reply:
Done.